src/usb/uf2/ghostfat.c

```diff
@@ -363,7 +363,12 @@ void read_block(uint32_t block_no, uint8_t *data) {
                 bl->targetAddr = addr;
                 bl->payloadSize = UF2_FIRMWARE_BYTES_PER_SECTOR;
                 bl->flags = UF2_FLAG_FAMILYID;
+
+#if defined(CFG_UF2_BOARD_APP_ID) && CFG_UF2_BOARD_APP_ID
```

Comment:
yeah, should always return the family = BOARD_ID here.

Reply:
Ok, fixed with commit aa8cd83.